proposals/Cargo.toml

```diff
@@ -10,13 +10,13 @@ repository = "https://github.com/webb-tools/webb-rs"
 license = "Apache-2.0"
 
 [dependencies]
-scale-codec = { package = 'parity-scale-codec', version = '3.0.0', default-features = false, optional = true, features = ["derive", "max-encoded-len"]}
+scale-codec = { package = 'parity-scale-codec', version = '3.0.0', default-features = false, optional = true, features = ["derive", "max-encoded-len"] }
 scale-info = { version = "2.1.1", default-features = false, optional = true }
 
 num-traits = { version = "0.2.8", default-features = false }
 typed-builder = { version = "0.10.0", default-features = false, optional = true }
 tiny-keccak = { version = "2.0.0", features = ["keccak"] }
-cosmwasm-std = { version = "1.0.0", default-features = false, optional = true }
+cosmwasm-std = { version = "1.0.0", default-features = false, optional = true, git = "https://github.com/CosmWasm/cosmwasm.git" }
```

Comment:
once https://github.com/CosmWasm/cosmwasm/pull/1374 publish a new version of cosmwasm-std we will remove the usage of git repo here.